pandas/tests/sparse/test_series.py

```diff
@@ -1360,4 +1360,4 @@ def test_numpy_func_call(self):
                  'argmin', 'argmax', 'min', 'max']
         for func in funcs:
             for series in ('bseries', 'zbseries'):
-                getattr(np, func)(getattr(self, series))
+                getattr(np, func)(getattr(self, series)._values)
```

Comment:
Isn't it the point of this test to test the numpy function call on a *Series* object ? If so, this change undoes the purpose of the test